wp-content/themes/workingnyc/views/employ-nyc-landing-page.twig

```diff
@@ -39,9 +38,9 @@
           {% for section in sections %}
```

Comment:
- [ ] Total padding between the header and the first section title should be 56 px. Change `<div class="py-4">` to `<div class="py-2">` to fix that (since there's already 40px of padding from the sticky)